<ui version="4.0" >
 <class>AddUserIDDialog</class>
 <widget class="QDialog" name="AddUserIDDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add New User-ID</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Real name:</string>
     </property>
     <property name="buddy" >
      <cstring>nameLE</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="nameLE" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;EMail address:</string>
     </property>
     <property name="buddy" >
      <cstring>emailLE</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="emailLE" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Co&amp;mment (optional):</string>
     </property>
     <property name="buddy" >
      <cstring>commentLE</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="commentLE" />
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>This is how the new User-ID will be stored in the certificate:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="errorLB" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>114</red>
             <green>114</green>
             <blue>114</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QLabel" name="resultLB" >
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddUserIDDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>236</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddUserIDDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddUserIDDialog</receiver>
   <slot>slotUserIDChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>414</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel" >
     <x>448</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddUserIDDialog</receiver>
   <slot>slotUserIDChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>412</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>450</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commentLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddUserIDDialog</receiver>
   <slot>slotUserIDChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>402</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel" >
     <x>446</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotUserIDChanged()</slot>
 </slots>
</ui>
